Alvind0.CodingTracker/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Alvind0.CodingTracker.Controllers;
+using Alvind0.CodingTracker.Data;
+using Alvind0.CodingTracker.Views;
+using Microsoft.Extensions.Configuration;
+
+var configuration = new ConfigurationBuilder()
+    .SetBasePath(Directory.GetCurrentDirectory())
+    .AddJsonFile("//Alvind0.CodingTracker//Properties//appsettings.json")
```

Comment:
🟡 **Hardcoded**

❓ Think of ways you could dynamically do this rather than hardcoded paths!